app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -635,7 +636,6 @@ public function buildCart($quote, $items, $multipage)
         $productMediaConfig = Mage::getModel('catalog/product_media_config');
         $cartSubmissionData = array(
             'order_reference' => $quote->getId(),
-            'display_id'      => $quote->getReservedOrderId(),
```

Comment:
Let's keep this. 99% of case it'll be accurate. If not, it can be updated via hook.

Reply:
I love this comment if it means the hook for updating this is on its way!!!!